src/Lightning.ts

```diff
@@ -308,7 +308,7 @@ export const LightningMixin = (superclass) =>
             const lightningLoggerOnUs = lightningLogger.child({ onUs: true, fee: 0 })
 
             if (await this.user.limitHit({ on_us: true, amount: tokens })) {
-              const error = `Cannot transfer more than ${this.config.limits.onUsLimit()} sats in 24 hours`
+              const error = `Cannot transfer more than ${this.config.limits.onUsLimit} sats in 24 hours`
```

Comment:
Perhaps we should refactor the naming and make it `this.config.limits.onUs` and get rid of the `Limit` suffix. The same can be done for the `withdrawalLimit` and for the generic limits as well since we're already accessing them by the `.limits.*` scope

Reply:
That makes sense, I'll check it out :+1:

Reply:
Yeah lets prefer flat structure from the POV of the calling code and also ignore the layout in `default.yml` as that would require changes to the provisioning logic.

Reply:
I'm just not getting into this refactor and there's one downside I'm seeing where renaming the property to something like `onUs` makes a bit trickier to grep for instances of it in the codebase since `onUS` (and later `intraLedger`) is more generically used in other places.

Is this is a tradeoff we'd like to consider, or should I go ahead with dropping `Limit` from places like these?

---

**Here's the current state of isolating the property for e.g.:**

![image](https://user-images.githubusercontent.com/17693119/127889907-2638dbb0-bb15-4e5a-a788-5d1011fa212c.png)